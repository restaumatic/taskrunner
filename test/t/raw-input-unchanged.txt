# no toplevel

go() {
  taskrunner -n mytask bash -e -c '
    snapshot --raw "${input}"
    echo "Expensive computation, input is: ${input}"
  '
}

export input=lol
go
go
