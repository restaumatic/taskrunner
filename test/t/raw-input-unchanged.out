-- output:
[mytask] info | Inputs changed, running task
[mytask] stdout | Expensive computation, input is: lol
[mytask] info | success
